packages/xchain-thorchain/src/client.ts

```diff
@@ -39,9 +39,9 @@ import {
   getDefaultExplorerUrl,
 } from './util'
 
-const MSG_SEND = 'send'
-const MSG_DEPOSIT = 'deposit'
-const MAX_TX_COUNT = 100
+export const MSG_SEND = 'send'
+export const MSG_DEPOSIT = 'deposit'
+export const MAX_TX_COUNT = 100
```

Comment:
Pls move these ^ to `src/util.ts` (where we have other exported constants defined=

Reply:
done 

https://github.com/xchainjs/xchainjs-lib/pull/306/commits/b61a74dfdbecf6d53e5e4f3bcd0e6b19bcc2c0cd